docs/automated-testing/ui-testing/teams-tests.md

```diff
@@ -0,0 +1,235 @@
+# Automated UI Tests for a Teams application
```

Comment:
Please notice these added words are causing the linter to break:
Unknown word (teamsfx)    

Unknown word (XVFB)       

Unknown word (framebuffer) 

Unknown word (Xvfb)        

Unknown word (XVFB)        

Unknown word (Xvfb)        

Unknown word (inprivate)   

Unknown word (Automator)   

Unknown word (Playstore)   

Unknown word (Automator)   

Unknown word (Automator)   

Unknown word (avds)        

Unknown word (runing)      

Unknown word (Automator)   

Unknown word (Automator)   

Unknown word (printscren)  

Unknown word (webdriverio) 

Unknown word (wdio)

Can you please add those to the .cspell?